forest/cli/src/cli/send_cmd.rs

```diff
@@ -7,9 +7,81 @@ use forest_json::message::json::MessageJson;
 use forest_rpc_client::{mpool_push_message, wallet_default_address};
 use fvm_shared::{address::Address, econ::TokenAmount, message::Message, METHOD_SEND};
 use num::BigInt;
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
 
 use super::{handle_rpc_err, Config};
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct FILAmount {
+    pub value: TokenAmount,
+}
+
+/// `FILAmount::from_str` adds support for parsing a string (input to the
+/// 'amount' field of the the `send` command) with all units supported by forest
+/// wallet (`attoFIL`, `femtoFIL`, `picoFIL`, `nanoFIL`, `microFIL`, `milliFIL`,
+/// and `FIL`) and converting the amount into an attoFIL `TokenAmount`. To match
+/// the current behavior in Lotus, the default units (if no units are specified
+/// in the command line) are `FIL`.
```

Comment:
This documentation is for the `form_str` function and it should just explain what the function does. In other words, the function doesn't "add support". The function just parses a string in a certain way.